sys/net/gnrc/network_layer/ipv6/nib/_nib-6ln.c

```diff
@@ -238,7 +239,7 @@ void _handle_rereg_address(const ipv6_addr_t *addr)
 }
 
 #if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_MULTIHOP_P6C)
-_nib_abr_entry_t *_handle_abro(const sixlowpan_nd_opt_abr_t *abro)
+_nib_abr_entry_t *_handle_abro(sixlowpan_nd_opt_abr_t *abro)
```

Comment:
I'd rather prefer to do the determination of `ltime` in `_handle_rtr_adv` as well, instead of this API change and introducing additional side-effects. E.g. by providing a function

```C
static inline uint16_t gnrc_sixlowpan_nd_opt_get_ltime(const sixlowpan_nd_opt_abr_t *abro)
{
    uint16_t ltime = byteorder_ntohs(abro->ltime);
    return (ltime == 0) ? SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT : ltime;
}
```

Reply:
done